sample-xml/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
Could we apply the same changes as in the first pull request? 

Reference: https://github.com/nimblehq/android-templates/pull/323/files#diff-3c8e8e5bb862d813f233c08097616ba081b3d4fe693fa46c37d73c8596f541adR1-R6

Reply:
@Tuubz It is the same changes, just a bit different with ordering. I don't think it is necessary 🙏